src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -108,4 +129,104 @@ private void addListingToBusinesses(PlacesSearchResult place) {
     allBusinesses.add(new Listing(name, formattedAddress, 
           placeLocation, rating, photos, types));
   }
-}
\ No newline at end of file
+
+  public void checkNumberOfLocationsOfBusiness() {
```

Comment:
This method initiates a few different checks to verify a big business. Please enumerate those with a comment here. It may also be worth it to change the name of this method appropriately. It's doing more than checking the number of locations.

Reply:
i changed the name of the function.